src/app/pages/license/license.hbs

```diff
@@ -0,0 +1,206 @@
+            </p>
+
+            <h3>How do I control what changes are made to my modules?</h3>
+            <p>
+                Only persons who have the maintainer role on your module have permission
+                to modify and publish an updated version of your module. As the author
+                of a module, you determine who has the maintainer role on your module.
+                The Creative Commons Attribution license does allow another person to
+                make an <a href="../authoring/expressedit">adaptation</a> of any content
+                published in our repository. An adaptation is a derived copy of a module
+                that is modified and published as a new module under the new author's
+                name. Your original module is not changed and the adaptation contains
+                a statement of attribution to you and a link to your original module.
+            </p>
+
+            <h3>But wait, I thought the license gave people the right to make their own adaptations of my work?</h3>
+            <p>
+                True, another author can indeed make an adaptation of your work according
+                to the license. However, if you determine that someone has created an
+                adaptation that is incorrect, or contradictory with your own, or worse,
+                is defamatory or otherwise offensive, the license gives you the legal
+                right to ask that person to remove your name from the module -- but not
+                to demand that he or she change it as you wish. One way to avoid creation
+                of such adaptations is to invite these other authors into your workgroup
+                and develop a dialog with them as collaborators or co-authors.
+            </p>
+
+            <h3>What prevents some dishonest person from stealing my works and claiming it as theirs?</h3>
+            <p>
+                Nothing except the copyright law. The Creative Commons license is based
+                on existing copyright law and your works have the same legal protections
+                and legal standing as a copyrighted work that is published outside of
+                our repository. You may actually run less risk using our repository than
+                simply putting your work on your own Web page, because the license is
+                clearly visible and clearly explains the uses people may legally make
+                of your work. Strictly speaking, fraud and plagiarism are not the subject
+                of copyright law, but copyright law is a good first defense.
+            </p>
+
+            <h3>Can people make money off of my work?</h3>
+            <p>
+                It is possible, but to be effective at all the license we use must allow
+                some "commercial" uses. Consider the situation in which a printing service,
+                such as FedEx Kinko's℠, offers to print your work so that your students
```

Comment:
I think this should be changed to "local printing service". The Kinko's name is no longer used.